azure-devops-sample/README.md

```diff
@@ -115,8 +115,6 @@ We can now test out our new connection!
 
 ## DevOps
```

Comment:
Since there's nothing here now can you go ahead and remove this header entirely?

Reply:
I don't think we should; this is a high level heading similar to `## Local` above - all the headings below this fall into the "DevOps" category for deployment, similar to how every heading under "Local" falls into local deployment

Reply:
Oops I misread it, you're totally correct. Merging now. Thanks for updating :)

Reply:
Thanks for the fast response :D